### PR TITLE
Add epsilon to normalize computation to prevent zero division.

### LINKED ISSUE #71: normalize has no epsilon in the denominator
The preprocessing functions image normalization, which z-score each channel in each image, don't have an epsilon in the denominator of this calculation. For images with very low standard deviation, this causes the values to blow up.

This caused weird bugs during training, which were resolved if I modified the above function to have a constant factor of 0.001 in the denominator of the z-score calculation

_Originally posted in vanvalenlab/deepcell-tf#445_

### PR DESCRIPTION
**Pull request by willgraf**
`processing.normalize` divides the image's mean value by its standard deviation. This PR adds a very small `epsilon` to the standard deviation to prevent any `NaN` issues when the standard deviation is 0.

A default value of `1e-07` is used as that is the default value found in `keras.backend.epsilon()`.

Fixes #71

### CODE REVIEW

**Review by ngreenwald — APPROVED** [review]
Awesome, thanks
